src/main/resources/db/migration/V2_24__add_prison_exclude_dates.sql

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE prison_exclude_date
+(
+    id                  serial           NOT NULL PRIMARY KEY,
```

Comment:
is the id needed? we could make prisonId and exclude_date combination as Primary Key.

Reply:
I think it the id is needed as composite primary keys cause problems - what do you do if a prison changes it's code?

so much better to always use surrogates as primary keys

Reply:
Agree with rich

Reply:
Sure.. will mark conversation as resolved.